lib/test/apyfixed/test_arithmetic.py

```diff
@@ -1,26 +1,27 @@
-from apytypes import APyFixed
+from apytypes import APyFixed, APyCFixed
 
 import math
 import pytest
 
 
-def test_arithmetic_operations():
-    a = APyFixed.from_float(-1.25, int_bits=2, frac_bits=4)
-    b = APyFixed.from_float(0.75, int_bits=2, frac_bits=3)
-    assert (a * b).is_identical(APyFixed.from_float(-0.9375, int_bits=4, frac_bits=7))
+@pytest.mark.parametrize("APyFix", [APyFixed, APyCFixed])
```

Comment:
```suggestion
@pytest.mark.parametrize("fixed_type", [APyFixed, APyCFixed])
```
Reading the code, something like that will make it clearer.